Loader/SymfonyFixturesLoader.php

```diff
@@ -72,6 +78,47 @@ protected function createFixture($class)
         return $this->loadedFixtures[$class];
     }
 
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @param string $set
+     *
+     * @return array $fixtures
+     */
+    public function getFixtures($set = null)
+    {
+        $fixtures = parent::getFixtures();
+
+        if ($set) {
+            $mapping = $this->setsFixtureMapping;
+            $fixtures = array_filter(
+                $fixtures,
+                function ($fixture) use ($mapping, $set) {
```

Comment:
Instead of passing both `$mapping` and `$set` why not just pass one variable containing `$mapping[$set]`?
That would also give you the option to return early without looping through all the fixtures if `$mapping[$set]` is empty or does not exist.